app/controllers/helm_infrastructures_controller.rb

```diff
@@ -43,6 +43,11 @@ def update
         "to_attributes" => @data_attributes.slice(:helm_cluster_template_id, :override_values, :is_active, :use_k8s_kibana)
       }
       broadcast(:app_group_updated, @helm_infrastructure.app_group.id)
+
+      if Figaro.env.ARGOCD_ENABLED == 'true'
+        response = ARGOCD_CLIENT.create_application(@helm_infrastructure.cluster_name, @helm_infrastructure.values, Figaro.env.argocd_default_destination_name)
```

Comment:
we need to do smth with the response, if it's not returning 2xx, we need to append the error to the `flash[:messages]`

Reply:
updated